src/main/java/org/osiam/resources/controller/GroupController.java

```diff
@@ -117,4 +131,13 @@ public void delete(@PathVariable final String id) {
         headers.setLocation(location);
         return new ResponseEntity<>(group, headers, status);
     }
+
+    private Set<String> extractAttributes(String attributesParameter) {
+        List<String> strings = Lists.newArrayList("schemas");
```

Comment:
Why is this a `List` if you convert it to a set in the end anyways? Shouldn't it be a `Set` und just get returned?